src/components/navigation.component.js

```diff
@@ -1,21 +1,21 @@
 import { html, LitElement, css } from "lit-element";
 import { cssStyles } from "../styles/cssStyles";
+import {
+  DEFAULT_IMG as defaultImg,
+  DEFAULT_NAME as defaultName
+} from "../constants/defaults.config";
 class Navigation extends LitElement {
   constructor() {
     super();
-    this.username = "username";
-    this.userImage=null;
-    this.isToken =
-      window.localStorage.getItem("token") === null ||
-      window.localStorage.getItem("token") === ""
-        ? false
-        : true;
+    this.username = defaultName;
+    this.userImage = defaultImg;
+    this.isToken = window.localStorage.getItem("token") ? true : false;

```

Comment:
Lets split logic out from the constructor. You can make a separate util to handle token and so on,